src/data/scripts/grade_buildings.py

```diff
@@ -439,14 +446,15 @@ def calculate_weighted_average(graded_df: pd.DataFrame) -> pd.Series:
 
     return weighted_average
 
-def grade_buildings(building_data):
+def grade_buildings():
+    df_historical = pd.read_csv(data_in_file_historical_path)
+
     # Generate grades for all years for GHG Intensity and Energy Mix:
     graded_df = grade_ghg_intensity_energy_mix_all_years(
-        building_data=building_data,
+        building_data=df_historical,
```

Comment:
This and line 450 are the primary change, making building grades apply to the historical data